### PR TITLE
feat(client): add adapter/cloudflare-d1 (empty)

### PR DESCRIPTION
**Pull request by Druue**
contributes https://github.com/prisma/team-orm/issues/522

Closes https://github.com/prisma/team-orm/issues/831

### CODE REVIEW

**Comment by Jolg42** [comment]
Fails with `Error: Cannot find module '/home/runner/work/prisma/prisma/packages/adapter-d1/helpers/build.ts'`

Let's merge once the tests and publishing passes and make a new PR for the adapter logic

**Comment by Jolg42** [comment]
Publishing to integration works: https://github.com/prisma/prisma/actions/workflows/release-ci.yml?query=branch%3Aintegration%2Fcloudflare-d1

https://www.npmjs.com/package/@prisma/adapter-d1

File: packages/adapter-d1/src/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const TEMP = 42
```
Comment (SevInf) [inline packages/adapter-d1/src/index.ts:1]:
```suggestion
export const ANSWER = 42
```

**Review by SevInf — APPROVED** [review]

File: packages/adapter-d1/src/index.ts (hunk 1 above)
Comment (Jolg42) [reply packages/adapter-d1/src/index.ts:1]: :trollface: Obviously, we also need a comment mentioning https://en.wikipedia.org/wiki/42_(number)

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**